packages/terra-notification-dialog/src/terra-dev-site/doc/example/NotificationDialogWithFocus.jsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import Button from 'terra-button';
 import { withDisclosureManager, disclosureManagerShape } from 'terra-disclosure-manager';
-import Popup from 'terra-popup/lib/Popup';
-import Placeholder from 'terra-doc-template/lib/Placeholder';
-import NotificationDialog, { NotificationDialogVariants } from '../../../NotificationDialog';
+import Popup from 'terra-popup';
+import Placeholder from 'terra-doc-template';
```

Comment:
Due to this change in Import the placeholder is not getting displayed in the popup . So I have a small PR #897 to revert it.